site_scons/site_tools/roc/pkgconfig.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+def FindPkgConfig(env, toolchain):
+    if env.HasArg('PKG_CONFIG'):
+        return True
+
+    if toolchain:
+        if env.Which(toolchain + '-pkg-config'):
+            env['PKG_CONFIG'] = toolchain + '-pkg-config'
+            return True
+        if env.Which('PKG_CONFIG_PATH') \
+            or env.Which('PKG_CONFIG_LIBDIR') \
+            or env.Which('PKG_CONFIG_SYSROOT_DIR'):
+            if env.Which('pkg-config'):
+                env['PKG_CONFIG'] = 'pkg-config'
+                return True
+        return False
+
+    if env.Which('pkg-config'):
```

Comment:
I guess this should be:

```
if not toolchain and env.Which('pkg-config'):
```

Reply:
here `toolchain`will be False because there is a `return False` in the end of above if toolchain check?

Reply:
Ah, I see, OK.